Makefile

```diff
@@ -1,5 +1,5 @@
 test:
-	python run-tests.py
+	py.test --doctest-modules testsuite jinja2
```

Comment:
Just run `py.test`, so the example tests are run too (these need some Python 3 fixes)